common/config/kloudmate.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import (
+	"errors"
+
+	"github.com/odigos-io/odigos/common"
+)
+
+const (
+	KloudMateEndpoint = "KLOUDMATE_ENDPOINT"
+)
+
+var (
+	ErrorKloudMateEndpointMissing = errors.New("KloudMate is missing a required field (\"KLOUDMATE_ENDPOINT\"), KloudMate will not be configured")
+)
+
+type KloudMate struct{}
+
+func (j *KloudMate) DestType() common.DestinationType {
+	return common.KloudMateDestinationType
+}
+
+func (j *KloudMate) ModifyConfig(dest ExporterConfigurer, currentConfig *Config) error {
+	config := dest.GetConfig()
+	uniqueUri := "kloudmate-" + dest.GetID()
+
+	url, exists := config[KloudMateEndpoint]
+	if !exists {
+		return ErrorKloudMateEndpointMissing
+	}
+
+	endpoint, err := parseOtlpHttpEndpoint(url)
```

Comment:
this function will allow unencrypted endpoints.
consider adding a parameter to require the scheme to be `https` and fail for `http`